### PR TITLE
Adding Splash and Upgrade screens

### PR DESCRIPTION
**Pull request by joroca**

### CODE REVIEW

File: gecosws_config_assistant/util/GtkAptProgress.py [hunk 1]
```diff
@@ -0,0 +1,391 @@
+import apt
+import apt_pkg
+import time
+import os
+import gi
+gi.require_version("Gtk", "3.0")
+gi.require_version("Vte", "2.90")
```
Comment (alfem) [inline gecosws_config_assistant/util/GtkAptProgress.py]: This line triggers an error: "ValueError: Namespace Vte not available for version 2.90"

**Review by alfem — CHANGES_REQUESTED** [review]

File: gecosws_config_assistant/util/GtkAptProgress.py (hunk 1 above)
Comment (joroca) [reply gecosws_config_assistant/util/GtkAptProgress.py]: Fixed. Support for gir1.2-vte-2.91

File: gecosws_config_assistant/controller/MainMenuController.py [hunk 2]
```diff
@@ -295,6 +296,7 @@ def checkForUpdates(self):
             apt_progress.show_terminal(True)
             try:
                 cache.commit(apt_progress.acquire, apt_progress.install)
+                time.sleep(5)
```
Comment (alfem) [inline gecosws_config_assistant/controller/MainMenuController.py]: What is this arbitrary sleep?

Comment (joroca) [reply gecosws_config_assistant/controller/MainMenuController.py]: This instruction avoids going from one screen to another too quickly for the user to read the update information

Comment (alfem) [reply gecosws_config_assistant/controller/MainMenuController.py]: I am not sure about a fixed time sleep. If you are showing info that must be read, a confirm button is needed. If it is just something you can oversee, then this sleep is not needed at all.

Comment (joroca) [reply gecosws_config_assistant/controller/MainMenuController.py]: Sleep command removed.

**Pull request merged**